.github/pull_request_template.md

```diff
@@ -1,13 +1,25 @@
-# Breaking Changes
+# Description
 
-N/A
+Please include a brief summary of the change.
```

Comment:
Would it make sense to have an optional backlink to the relevant Linear issues here?
```suggestion
Please include a brief summary of the change.

Linear issue: APKT-xxx
```

Reply:
+1 Ideally every PR should have been tracked on Linear and we should be getting the branch name from the Linear, so it'll automatically be linked to Linear and the ticket will be moved to next stage when the PR is merged.